front/src/applications/operationalStudies/views/Project.tsx

```diff
@@ -62,8 +63,18 @@ export default function Project() {
     [urlProjectId]
   );
 
-  const [deleteStudyV2] =
-    osrdEditoastApi.endpoints.deleteProjectsByProjectIdStudiesAndStudyId.useMutation();
+  const {
+    selectedItemIds: selectedStudyIds,
+    setSelectedItemIds: setSelectedStudyIds,
+    itemIds: studiesList,
+    setItemIds: setStudiesList,
+    toggleSelection: toggleStudySelection,
+    deleteItems: handleDeleteStudy,
+  } = useMultiSelection<StudyWithScenarios>((studyId) => {
+    if (selectedStudyIds.length > 0 && projectId && studyId) {
+      deleteStudyV2({ projectId, studyId });
+    }
```

Comment:
I've been thinking since the last time we did this together and I find better to make the check at the time where the delete is called in other words, where the `handleDeleteStudy` is called as follow : 

```typescript
  const handleDeleteStudy = () => {
    if (selectedStudyIds.length > 0 && projectId) {
      deleteItems();
    }
  };
```

Reply:
I feel like I have to check if projectId is defined before the deleteStudyV2, even if I'm checking it when handleDeleteStudy is called ?